src/test/java/org/fundacionjala/coding/oscar/Kata6Test.java

```diff
@@ -0,0 +1,49 @@
+package org.fundacionjala.coding.oscar;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Administrator on 8/29/2017.
+ */
+public class Kata6Test {
```

Comment:
to increase the code coverage you should test when the checksum is 0.
for instance.
```java
 assertTrue(eanValidator.validate("9783827317100"));
```